### DISCUSSION TITLE
We have set up our own colors in the config but tailwind is still allowing the use of their default colors. Can this be removed?

### DISCUSSION THREAD

**Discussion by CarelessCourage** [tailwindlabs/tailwindcss#10900] [Help] [answered]
I would like to strictly dictate what colors are availabe for our devs to use. We have moved over to a scemantic color strategy and we dont need any direct colors like you see in this screenshot. It would be really nice if we could just strip these away entierly and not even have the option to pick any other colors than the once defined in the config. Is this possible? 

![image](https://user-images.githubusercontent.com/12764398/228538477-accb94ce-07bd-4dc3-8e69-cb47918ef6c5.png)

**Comment by wongjn** [comment, marked as answer]
As per [the documentation](https://tailwindcss.com/docs/customizing-colors#disabling-a-default-color), use `theme.colors` instead of `theme.extend.colors`:
```js
module.exports = {
  theme: {
    colors: {
      transparent: 'transparent',
      current: 'currentColor',
      foo: {
        // …
      },
      bar: {
        // …
      },
      baz: {
        // …
      },
    },
  },
};
```

**Reply by CarelessCourage** [reply]
Ah, perfect. I was not the one on our team to originally set up the config so I didn't even realize we were extending. That make sense. Thanks
